packages/scene-composer/src/components/panels/__tests__/SettingsPanel.spec.tsx

```diff
@@ -5,28 +5,69 @@ import { SettingsPanel } from '..';
 import { useStore } from '../../../store';
 import { setFeatureConfig } from '../../../common/GlobalSettings';
 import { COMPOSER_FEATURES } from '../../../interfaces';
+import { mockProvider } from '../../../../tests/components/panels/scene-components/MockComponents';
 
-jest.spyOn(React, 'useContext').mockReturnValue('sceneComponserId' as any);
+jest.mock('../scene-settings', () => {
+  const originalModule = jest.requireActual('../scene-settings');
+  return {
+    ...originalModule,
+    SceneDataBindingTemplateEditor: 'SceneDataBindingTemplateEditor',
+    SceneTagSettingsEditor: 'SceneTagSettingsEditor',
+  };
+});
 
 jest.mock('../CommonPanelComponents', () => ({
   ...jest.requireActual('../CommonPanelComponents'),
   ExpandableInfoSection: (props) => <div data-mocked='ExpandableInfoSection' {...props} />,
 }));
 
 describe('SettingsPanel contains expected elements.', () => {
-  [{ [COMPOSER_FEATURES.TagResize]: true }, { [COMPOSER_FEATURES.TagResize]: false }].forEach((featureConfig) => {
-    it(`SettingsPanel contains expected elements for features ${featureConfig} `, async () => {
-      setFeatureConfig(featureConfig);
+  it('should contains default elements in editing mode.', async () => {
```

Comment:
NON-BLOCKING: Super minor typo here "should contain default..."

Looks like it got copied through the file so it's in multiple places.